job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -215,10 +113,12 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
     }
 
     val pb = Process(cmdString)
-    val pio = new ProcessIO(_ => (),
-                        stdout => scala.io.Source.fromInputStream(stdout)
-                          .getLines.foreach(println),
-                        stderr => scala.io.Source.fromInputStream(stderr).getLines().foreach(println))
+    val pio = new ProcessIO(
+      (_) => (),
```

Comment:
Yeah, I don't care too much either, like I said, mostly I hold that opinion from ES6 javascript where parens around single arg functions is becoming the standard. So if we want to do it one way, thats fine with me, but would be nice if scalastyle handled it :)